### ISSUE TITLE
Can't add package in XCode 15.1.

### ISSUE THREAD

**Issue by TheHmmka** [LottieFiles/dotlottie-ios#9]
Hello!

I constantly get next error and can't use this library in my project.

<img width="940" alt="Screenshot 2023-12-21 at 12 35 47 PM" src="https://github.com/LottieFiles/dotlottie-ios/assets/468660/0d696412-1c31-4d1e-80d0-8ebc48227bf9">

**Comment by samuelOsborne** [comment]
@TheHmmka Whats the version of the targets your targeting? macOS, iOS etc..

**Comment by TheHmmka** [comment]
> @TheHmmka Whats the version of the targets your targeting? macOS, iOS etc..

iOS started from 14.0

**Comment by samuelOsborne** [comment]
hi @TheHmmka, the minimum deployment for this package is iOS 16

**Comment by samuelOsborne** [comment]
@TheHmmka Please pull, works for ios 14 now 🥳

**Comment by TheHmmka** [comment]
> @TheHmmka Please pull, works for ios 14 now 🥳

The same issue =((

**Comment by ryo0126** [comment]
Hello!
I'm trying to use this library as [the previous repo](https://github.com/dotlottie/dotlottie-ios) is no longer maintained, but the same issue is happening in my project.

> dotlottie-ios could not be resolved:

![スクリーンショット 2024-01-30 17 35 29](https://github.com/LottieFiles/dotlottie-ios/assets/53012437/005cc1bc-ca42-4dab-8aa7-9681451e11a8)

How can I solve this issue?

# Environment

* Xcode 15.2 (15C500b)
* Minimum deployments of the target: iOS 15.0
* dotLottie v0.0.1

**Comment by samuelOsborne** [comment]
Let me check @ryo0126 And get back to you

**Comment by samuelOsborne** [comment]
hey @ryo0126 @TheHmmka I can't reproduce this error - I followed the steps in the README using the project link when adding the package to the ios project: https://github.com/LottieFiles/dotlottie-ios

could you please link a project recreating it ? Or email it to me at sam@lottiefiles.com

![image](https://github.com/LottieFiles/dotlottie-ios/assets/23125742/c992c3bd-627f-43c1-b913-3954b5cc79af)

![image](https://github.com/LottieFiles/dotlottie-ios/assets/23125742/b0e41a0e-e931-4472-a9d6-b3f2dd9c5157)

![image](https://github.com/LottieFiles/dotlottie-ios/assets/23125742/bf2b4120-61d0-4e03-a499-5122045d742a)

**Comment by ryo0126** [comment]
@samuelOsborne 
Thank you so much for your investigation.

I'm sorry, I checked it again and the cause was my setting of SSH.

I created a new project to test it and added dotLottie via SwiftPM, and the same error happened, and it seemed failed to clone the submodule, Thorvg, inside dotLottie.
So I removed all SSH keys from my Mac and GitHub and created/registered it again, resulting in success adding dotLottie via SwiftPM.

Thank you so much again.
